app/app.go

```diff
@@ -507,7 +507,7 @@ func NewWasmApp(
 
 	// The last arguments can contain custom message handlers, and custom query handlers,
 	// if we want to allow any custom callbacks
-	supportedFeatures := "iterator,staking,stargate"
+	supportedFeatures := "iterator,staking,stargate,cosmwasm_1_1"
```

Comment:
Should this be documented for the release? 

Also we started to enforce version check for Stargaze to prevent validators from using a different wasm library and avoid non-determinism  from nodes that don't upgrade correctly.

https://github.com/public-awesome/stargaze/blob/main/cmd/starsd/cmd/start.go

Not really sure if this is something that should also be part of wasmd.

Reply:
>  using a different wasm library and avoid non-determinism from nodes that don't upgrade correctly.

Not sure if I got that correct. Do you have validators that overwrite the wasmvm dependency? That would be hard to cover as they may overwrite other parts as well. Feel free to open an issue to discuss this further.

Reply:
> > using a different wasm library and avoid non-determinism from nodes that don't upgrade correctly.
> 
> Not sure if I got that correct. Do you have validators that overwrite the wasmvm dependency? That would be hard to cover as they may overwrite other parts as well. Feel free to open an issue to discuss this further.

Not overwrite but they often compile on one machine upload the binary to the remote/secured host and run it so it may be running with an old libwasm. It has caused consensus issues before, at least from our side we prevent nodes from running if the lib is not the right version as you can see the code I pointed. But yes it is a different discussion.

Reply:
Thanks for adding more context. I think this is a valid use case. I prefer static compilation but this is an issue indeed